pom.xml

```diff
@@ -113,12 +113,18 @@
       		<artifactId>bcprov-jdk16</artifactId>
       		<version>1.46</version>
     	</dependency>
-		
-		<dependency>
-			<groupId>com.jcraft</groupId>
-			<artifactId>jsch</artifactId>
-			<version>0.1.51</version>
-		</dependency>
+
+      <dependency>
+      	<groupId>com.jcraft</groupId>
+      	<artifactId>jsch</artifactId>
+      	<version>0.1.51</version>
```

Comment:
Would be better to depend on a plugin like Publish Over SSH to avoid binary compat issues